rezerwacje-front/src/views/Faculties/components/FacultyDialog/FacultyDialog.jsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import {
+  Grid,
+  Button,
+  TextField,
+  Typography,
+  DialogActions,
+  IconButton,
+} from '@material-ui/core';
+import { makeStyles, useTheme } from '@material-ui/styles';
+import CloseIcon from '@material-ui/icons/Close';
+import Dialog from '@material-ui/core/Dialog';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogTitle from '@material-ui/core/DialogTitle';
+
+import validateFacultyForm from './validateFacultyForm.js';
+import { useDialogForm } from 'common/utilities';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    margin: 0,
+    padding: theme.spacing(2),
+  },
+  closeButton: {
+    position: 'absolute',
+    right: theme.spacing(1),
+    top: theme.spacing(1),
+    color: theme.palette.grey[500],
+  },
+}));
+
+const FacultyDialog = (props) => {
+  const classes = useStyles();
+
+  const submit = () => {
+    const { id, name } = values;
+
+    fetch(props.url, {
+      method: props.httpMethod,
+      headers: { 'Content-Type': 'application/json' },
+      body: JSON.stringify({
+        id: id,
```

Comment:
When I try to add a new faculty, I get an error, because the `id` is null.

Reply:
Ok fixed that